.github/workflows/docs-version.yml

```diff
@@ -0,0 +1,75 @@
+name: Version Documentation on Tag
+
+on:
+  push:
+    tags:
+      - 'v*.*.*'  # Triggers on version tags like v1.0.0, v2.1.3, etc.
```

Comment:
We might want to reconsider using the `v` prefix. 

Unless we intend to publish parts of this repo as a Go library, a plain `x.y.z` version may communicate more clearly that the code isn’t meant for reuse. 

If someone really wants to depend on it, they can rely on Go pseudo-versions - with no support or guarantees. This is the approach Istio takes for its core components (which is a headache for consumer, but I do not see this happening here soon).

Thoughts?